Imagr/MainController.py

```diff
@@ -1676,8 +1676,22 @@ def eraseTargetVolume(self, name='Macintosh HD', format='Journaled HFS+', progre
         Erases the target volume.
         'name' can be used to rename the volume on reformat.
         'format' can be used to specify a format type.
+            'format' type of 'auto_hfs_or_apfs' will check for HFS+ or APFS
```

Comment:
Can you fix this formatting? You likely have a tab character instead of space or vis verse.

Reply:
Fixed